lib/utils/visUtil.hpp

```diff
@@ -583,9 +584,13 @@ class StatTracker {
     /**
      * @brief Create a ring buffer.
      *
+     * @param name Buffer name.
+     * @param unit Sample unit.
      * @param size The size of the ring buffer.
+     * @param is_optimized Flag of min/max optimization.
```

Comment:
I'd like slight more description of what this flag does.  e.g. what does this flag do, and why might you choose to use it (or not).   You could add this to the main body of the comment block after the `@brief` line, since there isn't a lot of space in the `@param` and it's an important part of how this class works.